src/rgw/rgw_coroutine.cc

```diff
@@ -94,6 +94,7 @@ void RGWCompletionManager::go_down()
   Mutex::Locker l(lock);
   for (auto cn : cns) {
     cn->unregister();
+    delete cn.get();
```

Comment:
should be `cn->put()`. Not sure why we haven't seen this before, @cbodley does this make sense?